kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/KieServicesClientImpl.java

```diff
@@ -123,6 +119,30 @@ private void init() {
 
     }
 
+    private List<String> getCapabilitiesFromServer() {
+        if (config.isJms()) {
+            if (config.isJmsTransactional()) {
```

Comment:
I'd recommend at removing this as it does not provide much value and should not be used with async handlers on kie services client.